src/components/job-common.js

```diff
@@ -161,3 +161,12 @@ export const workflowDetailsBreadcrumbSubtitle = (namespace, workspaceName, subm
     h3({ style: Style.elements.sectionHeader }, [`Workflow ${workflowId}`]),
   ]);
 };
+
+export const getTaskCost = (vmCost, startTime, endTime) => {
+  const currentEndTime = Date.parse(endTime) || Date.now();
+  const vmCostDouble = parseFloat(vmCost);
+  const startDateTime = Date.parse(startTime);
+
+  const elapsedTime = currentEndTime - startDateTime;
+  return parseFloat(((elapsedTime / 3600000) * vmCostDouble).toFixed(2));
```

Comment:
Should we round this up to one cent when the cost is great than 0?

Reply:
I think the best behavior here would be to round (up or down) to the nearest cent, and if that turns out to be 0.00 instead display `< $0.01`